kernel/Kconfig

```diff
@@ -9,6 +9,18 @@ module = KERNEL
 module-str = kernel
 source "subsys/logging/Kconfig.template.log_config"
 
+config KERNEL_WARN_ALLOC_INTERVAL
+	int "Interval of pending wait for event warnings"
+	default 1
+	range -1 60
+	depends on KERNEL_LOG_LEVEL_WRN
```

Comment:
This will make it only available if the kernel log level is exactly set to `WRN`, but not with `INF` or `DBG`...